docs/sources/dashboards/_index.md

```diff
@@ -1,11 +1,64 @@
 +++
-title = "Dashboards"
+title = "Dashboard overview"
 type = "docs"
+aliases = ["/docs/grafana/latest/features/dashboard/dashboards/"]
 [menu.docs]
 identifier = "dashboards"
-weight = 4
+weight = 100
 +++
 
 # Dashboard overview
 
-A *dashboard* is a set of one or more panels organized and arranged into one or more rows. Grafana ships with a variety of Panels. Grafana makes it easy to construct the right queries, and customize the display properties so that you can create the perfect dashboard for your need. Each panel can interact with data from any configured Grafana Data Source (currently Graphite, Prometheus, Elasticsearch, InfluxDB, OpenTSDB, MySQL, PostgreSQL, Microsoft SQL Server and AWS Cloudwatch).
\ No newline at end of file
+A _dashboard_ is a set of one or more panels organized and arranged into one or more rows. Grafana ships with a variety of Panels. Grafana makes it easy to construct the right queries, and customize the display properties so that you can create the perfect dashboard for your need. Each panel can interact with data from any configured Grafana Data Source (currently Graphite, Prometheus, Elasticsearch, InfluxDB, OpenTSDB, MySQL, PostgreSQL, Microsoft SQL Server and AWS Cloudwatch).
+
+## Dashboard UI
+
+<img src="/img/docs/v50/dashboard_annotated.png" class="no-shadow" width="700px">
+
+1. Zoom out time range
+2. Time picker dropdown. Here you can access relative time range options, auto refresh options and set custom absolute time ranges.
+3. Manual refresh button. Will cause all panels to refresh (fetch new data).
+4. Dashboard panel. Click the panel title to edit panels.
+5. Graph legend. You can change series colors, y-axis and series visibility directly from the legend.
+
+## Dashboard header
+
+Click the new Dashboard link on the right side of the Dashboard picker. You now have a blank Dashboard.
+
+<img class="no-shadow" src="/img/docs/v50/top_nav_annotated.png" width="580px">
+
+The image above shows you the top header for a Dashboard.
+
+1. Side menubar toggle: This toggles the side menu, allowing you to focus on the data presented in the dashboard. The side menu provides access to features unrelated to a Dashboard such as Users, Organizations, and Data Sources.
+2. Dashboard dropdown: This dropdown shows you which Dashboard you are currently viewing, and allows you to easily switch to a new Dashboard. From here you can also create a new Dashboard or folder, import existing Dashboards, and manage Dashboard playlists.
+3. Add Panel: Adds a new panel to the current Dashboard
+4. Star Dashboard: Star (or unstar) the current Dashboard. Starred Dashboards will show up on your own Home Dashboard by default, and are a convenient way to mark Dashboards that you're interested in.
```

Comment:
"...own Home Dashboard by default. It is a convenient way to mark Dashboards that you're interested in."